core/shared/src/main/scala/zio/logging/package.scala

```diff
@@ -54,6 +54,23 @@ package object logging extends LoggerLayers {
   def loggerName(value: String): ZIOAspect[Nothing, Any, Nothing, Any, Nothing, Any] =
     ZIOAspect.annotated(loggerNameAnnotationKey, value)
 
+  /**
+   * Append logger name aspect, by which it is possible to append a logger name to the current logger name.
+   * The new name is appended using a dot (.) as the delimiter.
+   *
+   * annotation key: [[zio.logging.loggerNameAnnotationKey]]
+   */
+  def appendLoggerName(value: String): ZIOAspect[Nothing, Any, Nothing, Any, Nothing, Any] =
+    new ZIOAspect[Nothing, Any, Nothing, Any, Nothing, Any] {
+      def apply[R, E, A](zio: ZIO[R, E, A])(implicit trace: Trace): ZIO[R, E, A] =
+        for {
+          annotations      <- FiberRef.currentLogAnnotations.get
```

Comment:
i think there is `ZIO.logAnnotations` function , which actually calling `FiberRef.currentLogAnnotations.get` but it seems to be better to use `ZIO.logAnnotations` here

Reply:
thanks, fixed.